modules/permutiveRtdProvider.js

```diff
@@ -222,10 +242,19 @@ function getCustomBidderFn (moduleConfig, bidder) {
  * @return {Object} Bidder function
  */
 function getDefaultBidderFn (bidder) {
+  const isPStandardTargetingEnabled = (data, acEnabled) => {
+    return (acEnabled && data.ac && data.ac.length) || (data.ssp && data.ssp.cohorts.length)
```

Comment:
@zshnr Hello!

I am seeing `data.ssp.cohorts` come back as `undefined` in some cases for our publishers prebid wrappers.

Here is an example:

<img width="1317" alt="image" src="https://user-images.githubusercontent.com/13972794/211613717-daa2673f-d94a-499c-a29e-3b64b75f48e1.png">


This causes an exception to be thrown and the permutive `makeSafe` function catches it but probably does not do all of the necessary stuff!

So probably just need to add a safe check here for making sure `cohorts` is defined before accessing `length`

Reply:
Seems like it is okay on the publisher site when the Perutive endpoint comes back with an actual data resonse:

<img width="1406" alt="image" src="https://user-images.githubusercontent.com/13972794/211617581-f604beae-0775-4921-aa18-ad683b709c99.png">

So probably not a huge deal, but just in case might be a good idea to add a safe check!

Reply:
Ah that is bizzare 🤔 . I was sure if the `ssp` property exists then it will have a `cohorts` property even if its empty.

Thanks for flagging I'll check internally. @robertrmartinez

Reply:
Hi @robertrmartinez , I have merged a fix for this :) 

To help us understand the impact of this, can you let me know which publisher websites you saw this on?

Thanks!